packages/calcite-components/src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -176,14 +250,19 @@ export class DatePickerMonth {
 
   @Listen("pointerout")
   pointerOutHandler(): void {
-    this.calciteInternalDatePickerMouseOut.emit();
+    this.calciteInternalDatePickerMonthMouseOut.emit();
   }
 
   //--------------------------------------------------------------------------
   //
   //  Lifecycle
   //
   //--------------------------------------------------------------------------
+
+  connectedCallback(): void {
```

Comment:
Is there another lifecycle hook you could use? It might be too early for this. This might be more of an issue with the `components` output build.